src/scripts/background.js

```diff
@@ -1435,6 +1435,10 @@ window.TogglButton = {
   },
 
   checkWorkDayEnd: async function () {
+    if (!TogglButton.$curEntry) {
+      return;
+    }
+
     const stopAtDayEnd = await db.get('stopAtDayEnd');
     const hasWorkdayEnded = await TogglButton.workdayEnded();
```

Comment:
This would return false from `workdayEnded` if `!TogglButton.$curEntry`, anyway. But also doesn't harm to also check here :)